mlflow/store/tracking/utils/sql_trace_metrics_utils.py

```diff
@@ -184,6 +189,31 @@ def _get_aggregation_expression(aggregation: MetricAggregation, db_type: str, co
             )
 
 
+def _get_assessment_numeric_value_column(json_column: Column) -> Column:
+    """
+    Extract numeric value from JSON-encoded assessment value.
+
+    Handles conversion of JSON primitives to numeric values:
+    - JSON true/false -> 1/0
+    - JSON numbers -> numeric value
+    - other JSON-encoded values -> NULL
+
+    Args:
+        json_column: Column containing JSON-encoded value
+
+    Returns:
+        Column expression that extracts numeric value or NULL for non-numeric values
+    """
+    return case(
```

Comment:
is it possible to get a `null` value ? (it is also a valid json value) , if yes we should add a case like `(json_column == "null", None),`